.cypress/cypress/integration/bathnes.js

```diff
@@ -39,3 +39,12 @@ it('uses the Curo Group housing layer correctly', function() {
     cy.pickCategory('Dog fouling');
     cy.contains('Maintained by Curo Group').should('be.visible');
 });
+
+it('handles code names with spaces without error', function() {
+    cy.visit('http://bathnes.localhost:3001/report/new?longitude=-2.368929&latitude=51.386969');
```

Comment:
The lat/long that you've used here will need to be added to `t/Mock/MapIt.pm`, which is the mocked version of MapIt that's used in the tests.

You can see the lat/long that's used by the test above here:

https://github.com/mysociety/fixmystreet/blob/6f35c67e76ef8efdbca3d2e6fdec17d1625cabbd/t/Mock/MapIt.pm#L68

So I think you can basically duplicate that line and change the lat/long to the ones you're using here.

Reply:
Or alternatively you could re-use the same lat/long as the Curo test, think that would also work.